package io.micro.core.exception

/**
 *@author Augenstern
 *@since 2023/10/11
 */
class RetryableException : RuntimeException() {
}